Automattic/jetpack
# Theme Tools: rm extra Social Menu added in Twenty Twenty theme

Fixes #20991

#### Changes proposed in this Pull Request:
We do not need an extra menu, Twenty Twenty already provides its own.

#### Jetpack product discussion

* No

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Before you check out this branch, go to Appearance > Themes
* Enable the Twenty Twenty theme
* Go to Customize > Menus
* Notice the 2 Social Menu locations.
* Create 2 different Social menus, and add different items to each.
* You'll notice that one of them is not displayed anywhere on the site.
* Check out this branch.
* Back in Appearance > Customize > Menus, there will only be one Social Menu available, and the menu that was visible before you checked out that branch is still visible.
